src/coreclr/hosts/inc/coreclrhost.h

```diff
@@ -126,7 +126,7 @@ CORECLR_HOSTING_API(coreclr_execute_assembly,
 //
 // Callback types used by the hosts
 //
-typedef bool(CORECLR_CALLING_CONVENTION BundleProbeFn)(const char* path, int64_t* offset, int64_t* size);
+typedef bool(CORECLR_CALLING_CONVENTION BundleProbeFn)(const char* path, int64_t* offset, int64_t* sizes);
```

Comment:
Does Windows not allow unicode characters in the path? Or is this really a byte stream?

Reply:
If I remember correctly - all runtime/host interactions use UTF8 strings.